src/components/AuthSecretKey.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { useState } from "react";
+import { Link, useHistory } from "react-router-dom";
+import { useDispatch } from "react-redux";
+import styled from "styled-components";
+import { Keypair } from "stellar-sdk";
+
+import { fetchAccount } from "ducks/account";
+
+const Warning = styled.div`
```

Comment:
We should follow the styled-components `El` naming convention strictly:

https://github.com/stellar/product-conventions/blob/master/STYLEGUIDE.md#naming-1